travis/before_install.sh

```diff
@@ -1,38 +1,25 @@
 #!/bin/bash
 
+set -e
+
 echo "Running before_install"
-echo "SUITE: ${SUITE}"
 
-"echo 'gem: --no-ri --no-rdoc' > ~/.gemrc"
+# Assert that the given SUITE is a known one
+case "$SUITE" in
+    rspec|rubocop|cucumber)
+        echo "SUITE: ${SUITE}"
+        ;;
+    *)
+        echo -e "Error: empty or unknown SUITE: $SUITE\n"
```

Comment:
Ah yeah, good catch. Didn't think of it since the line was already there previously and I just changed the content.